Sources/Twift+Tweets.swift

```diff
@@ -262,6 +262,13 @@ public struct DeleteResponse: Codable {
   public let deleted: Bool
 }
 
+/// A response object pertaining to list created
+public struct CreatedListResponse: Codable {
+    /// If the returned response object contains an id and the name of your List, you have successfully created the List.
```

Comment:
This doc comment will only appear when authors look up the `id` attribute; you should add doc comments specific to each property and add this comment as a description on the type itself